PorticoExportDom.inc.php

```diff
@@ -15,10 +15,10 @@
 
 class PorticoExportDom {
 	/** @var string DTD URL of the exported XML */
-	private const PUBMED_DTD_URL = 'http://dtd.nlm.nih.gov/archiving/3.0/archivearticle3.dtd';
+	private const PUBMED_DTD_URL = 'http://jats.nlm.nih.gov/archiving/1.2/JATS-archivearticle1.dtd';
```

Comment:
I've bumped the version to the latest stable revision, also I'm using the most permissive DTD (there are other 2).